templates/cloud/cloud-computing.html

```diff
@@ -0,0 +1,494 @@
+          <p><a href="/cloud/hybrid-cloud">More about hybrid clouds&nbsp;&rsaquo;</a></p>
+        </div>
+      </div>
+      <div class="col-4">
+        <div class="u-sv1 u-hide--small">
+          {{
+            image (
+            url="https://assets.ubuntu.com/v1/dd0465c8-Multi-Cloud.svg",
+            alt="Multi-Cloud",
+            width="113",
+            height="64",
+            hi_def=True,
+            loading="auto"
+            ) | safe
+          }}
+          <h4>Multi-cloud</h4>
+          <p>Multi-cloud (also referred to as multi cloud or multicloud) is a concept that refers to using multiple clouds from more than one cloud service provider at the same time. The term is also used to refer to the simultaneous running of bare metal, virtualised and containerised workloads.</p>
+          <p><a href="/cloud/multi-cloud">More about multi-cloud&nbsp;&rsaquo;</a></p>
+        </div>
+      </div>
+    </div>
+  </div>
+</section>
+<section class="p-strip u-no-padding--bottom">
+  <div class="row u-equal-height">
+    <div class="col-8">
+      <h2>Cloud computing models</h2>
+      <p class="u-sv3">Cloud computing services are usually available to end users in the form of three primary models. Those include infrastructure-as-a-service (IaaS), platform-as-a-service (PaaS) and software-as-a-service (SaaS). Some more specific use cases exist too, such as container-as-a-service (CaaS). However, in essence, they are a subset of the main three.</p>
+    </div>
+  </div>
+  <div class="row">
+    <div class="p-card col-4">
+      <h4>IaaS</h4>
+      <p>In the IaaS model, you provision resources. Those include the number of virtual CPUs (vCPUs), the amount of RAM, storage, etc. They come in the form of VMs or containers with a pre-installed operating system (OS). You manage everything up from there. IaaS is the most common cloud computing model as it allows for more freedom.</p>
+    </div>
+    <div class="p-card col-4">
+      <h4>PaaS</h4>
+      <p>In the PaaS model, you provision workloads. While you are still responsible for delivering application code and data management, the PaaS platform takes care of scheduling resources (usually containers) and manages them, including the OS, middleware and runtime. The PaaS model has never been widely adopted due to its overall complexity.</p>
+    </div>
+    <div class="p-card col-4">
+      <h4>SaaS</h4>
+      <p>In the SaaS model, you provision applications. They are deployed from pre-defined templates and can be configured according to your needs. Everything is managed by the cloud provider. Interest in the SaaS model is constantly increasing as it allows for full automation from the ground up.</p>
+    </div>
+  </div>
+  <div class="p-strip p-table--cloud-computing">
+    <table aria-label="Model comparison matrix" style="table-layout: auto">
+      <thead>
+        <tr>
+          <th>&nbsp;</th>
```

Comment:
```suggestion
          <td>&nbsp;</td>
```
As this doesn't contain a header it can be a td, the is best practice for accessibility